### PR TITLE
Test/626 oracle info

### LINKED ISSUE #626: Add Oracles info to Console V2
Follow on from https://github.com/vegaprotocol/frontend-monorepo/issues/625 This to add details about the oracle a market uses.

## Background

When someone proposed a market they also define 2 oracle specifications (or re-use an existing ones).
`oracleSpecForTradingTermination` is used to create an event to [terminate the market](https://github.com/vegaprotocol/specs-internal/blob/master/protocol/0043-MKTL-market_lifecycle.md#trading-terminated),
`oracleSpecForSettlementPrice` is used as the price to [settle](https://github.com/vegaprotocol/specs-internal/blob/master/protocol/0043-MKTL-market_lifecycle.md#settled)  positions that are still open.

```GraphQL
{
  markets{
    tradableInstrument{
      instrument{
        product{
          ...on Future{
            oracleSpecForSettlementPrice{
              id
            }
            oracleSpecForTradingTermination{
              id
            }
          }
        }
      }
    }
  }
}
```

These IDs reference an **oracle Specification**:
```GraphQL
{
  oracleSpecs{
    id
    status
    pubKeys
    createdAt
    updatedAt
    filters{
      key {
        name
        type
      }
      conditions {
        value
        operator
      }
    }
  }
}
```

Each specification is "Listening" to a transaction type **Oracle Submission**
See [WIP docs](https://deploy-preview-121--vega-docusaurus.netlify.app/docs/testnet/tutorials/oracles#3-submit-the-message-to-the-chain-1) Which might already be [live](http://docs.vega.xyz/docs/testnet/tutorials/oracles).

You can see all oracles submissions with
```GraphQL
{
  oracleData{
    data{
      name
      value
    }
    broadcastAt
    matchedSpecIds
	}
}
```

When an oracle submission matches a filter it creates a **Binding**.
You'll see a `broadcastAt` and the specID that it matches.

note: some names are [up for debate](https://github.com/vegaprotocol/vega/issues/5689).

## Design + UX requirements

I suggest that the market info at very least should include the `oracleSpecForTradingTermination` and `oracleSpecForSettlementPrice` IDs. At a minimum it could link to the block explorer that should show all the [raw oracle information](https://github.com/vegaprotocol/frontend-monorepo/issues/633).
But ideally Console would also show this information...

### PR DESCRIPTION
**Pull request by jtsang586**
# Related issues 🔗

Closes #626

# Description 

- Updates tests to include additional market info
- Added assertions for new withdrawal delay threshold text
- Change cypress browser to electron to try and improve intermittent failing tests

### CODE REVIEW

**Review by asiaznik — APPROVED** [review]

File: apps/trading-e2e/src/integration/market-info.cy.ts [hunk 1]
```diff
@@ -129,6 +126,21 @@ describe('market info is displayed', () => {
     validateMarketDataRow(2, 'Scaling Factor', '10');
   });
 
+  it('liquidity displayed', () => {
+    // cy.getByTestId(marketTitle).contains('Liquidity').last().click();
```
Comment (sam-keen) [inline apps/trading-e2e/src/integration/market-info.cy.ts:130]: Chore (non-blocking): Unless there's a compelling reason, might as well remove this commented-out code

File: apps/trading-e2e/src/integration/market-info.cy.ts [hunk 2]
```diff
@@ -148,6 +160,22 @@ describe('market info is displayed', () => {
       'Termination Oracle',
       'f028fe5ea7de3890962a05a7163fdde562629af649ed81b8c8902fafb6eef04f'
     );
+
+    cy.getByTestId(externalLink)
+      .should('have.attr', 'href')
+      .and('contain', 'https://explorer.fairground.wtf/oracles');
```
Comment (sam-keen) [inline apps/trading-e2e/src/integration/market-info.cy.ts:166]: Question (non-blocking): Should these urls be environment-aware? I'm wondering if we'd ever want to run these tests against mainnet

**Review by sam-keen — APPROVED** [review]

File: apps/trading-e2e/src/integration/market-info.cy.ts (hunk 1 above)
Comment (jtsang586) [reply apps/trading-e2e/src/integration/market-info.cy.ts]: Removed

File: apps/trading-e2e/src/integration/market-info.cy.ts (hunk 2 above)
Comment (jtsang586) [reply apps/trading-e2e/src/integration/market-info.cy.ts]: That's a valid point. I've loosened the assertion so it should still pass when if we ever want to run these tests on another environment.

**Pull request merged**